examples/relativity/relativity.py

```diff
@@ -131,7 +131,7 @@ def recalculate(self):
     def setAnimation(self, a):
         if a:
             self.lastAnimTime = pg.ptime.time()
-            self.animTimer.start(self.animDt*1000)
+            self.animTimer.start((self.animDt*1000))
```

Comment:
Should this be `self.animTimer.start(int(self.animDt*1000))` ?

Reply:
absolutely should be; not sure how I messed that up; I redid that bit of git history to save myself the embarrassment